extensions/db-diagram/src/controllers/mainController.ts

```diff
@@ -29,14 +31,31 @@ export default class MainController extends ControllerBase {
 		return new Promise<boolean>(async (resolve) => {
 			vscode.commands.registerCommand('db-diagram.new', async () => {
 
-				// ...
-				console.log('new diagram');
-
-
+				//Dummy Data
+				/*const serverName = 'My Server';
+				const dbName = 'My Database';
+				const dbInformation = 'This is a database about a company. There is 1 significant cluster';
+				const tables = ['Employee', 'Department', 'Manager'];*/
+
+				// Create and show a new webview
+				const panel = vscode.window.createWebviewPanel(
+					'dbDiagram', // Identifies the type of the webview. Used internally
```

Comment:
You don't need to comment on these, the JSDoc provides good enough context on them